### ISSUE TITLE
URI:s for StructureDefinitions

### ISSUE THREAD

**Issue by peturv** [commonprofiles-care/fhir#26]
To discuss in next meeting:

The currently suggested format for StructureDefinition.url for our profiles is `https://commonprofiles.care/StructureDefinition/_PROFILE_/v1`

This deviates somewhat from the normal handling of FHIR resources where the norm is `http(s)://SERVER/RESOURCE_TYPE/RESOURCE_ID` where the versioning is handled in the resource, by the _history or similar https://www.hl7.org/fhir/http.html#versioning

URI:s that point at the profiles are of the data type canonical(StructureDefinition) where versioning is handled with a pipe `|` character. https://www.hl7.org/fhir/datatypes.html#canonical

( It also differs from traditional URI versioning of REST API:s where the version is traditionally prepended in front of the resource type, for example: api.server.com/v1/RESOURCE_TYPE )

In short: I suggest removing the /v1 in favor of following the more FHIR-ish way of version handling the profiles.

**Comment by vjernelov** [comment]
@RebeckaHansson lets her techy team members review this change and see if they have any objections.

**Comment by niclas86** [comment]
@RebeckaHansson will look at this

**Comment by RebeckaHansson** [comment]
Due to hectic period currently I haven't been able to take the time with the people I need input from on this. Will work on it until next meeting.

**Comment by peturv** [comment]
Let me know if you want any involvement from me here in discussions.

From: RebeckaHansson ***@***.***>
Date: Friday, 30 September 2022 at 10:14
To: SvenskaIndustriProfiler/fhir ***@***.***>
Cc: Pétur Valdimarsson ***@***.***>, Author ***@***.***>
Subject: Re: [SvenskaIndustriProfiler/fhir] URI:s for StructureDefinitions (Issue #26)

Due to hectic period currently I haven't been able to take the time with the people I need input from on this. Will work on it until next meeting.

**Comment by niclas86** [comment]
@RebeckaHansson has booked a meeting for this next week

**Comment by niclas86** [comment]
Suggestion is to go forward with this, @peturv will you update profiles as this suggestion

**Comment by niclas86** [comment]
This is already in place.
